UPGRADE-2.8.md

```diff
@@ -238,6 +238,21 @@ Form
 Translator
 ----------
 
+ * Loading catalogues (file caching, resource loader, locale fallback) has been moved out of Translator and the class
+   `Symfony\Bundle\FrameworkBundle\Translation\Translator` was deprecated and will be removed in Symfony 3.0.
+
+   If you want to use the new dedicated translator service, you should either
+   explicitly define "translator" service.
```

Comment:
The `translator` service use the deprecated class `Symfony\Bundle\FrameworkBundle\Translation\Translator` and I was thought that may occurs a BC break but after rethinking about it I made another way to solve it.